### PR TITLE
docs: Add multimodal image reasoning tutorial with llama index

### PR DESCRIPTION
**Pull request by fjcasti1**
Closes https://github.com/Arize-ai/openinference/issues/561

### CODE REVIEW

File: tutorials/llama_index_multimodal_image_reasoning.ipynb [hunk 1]
```diff
@@ -0,0 +1,466 @@
+{
```
Comment (mikeldking) [inline tutorials/llama_index_multimodal_image_reasoning.ipynb:1]: <p>This feels like the title and instructions</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arize-ai/phoenix/pull/4210/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (mikeldking) [inline tutorials/llama_index_multimodal_image_reasoning.ipynb:1]: <p>I would combine this with the below, nix the TBDs and not use the warning symbol for llama-index</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arize-ai/phoenix/pull/4210/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

**Review by mikeldking — APPROVED** [review]

File: tutorials/llama_index_multimodal_image_reasoning.ipynb (hunk 1 above)
Comment (fjcasti1) [reply tutorials/llama_index_multimodal_image_reasoning.ipynb:1]: <p>Oh yes they were meant to be combined my bad</p>

Comment (fjcasti1) [reply tutorials/llama_index_multimodal_image_reasoning.ipynb:1]: <p>Its meant to be hehe</p>

**Pull request merged**